examples/vector_databases/elasticsearch/README.md

```diff
@@ -0,0 +1,30 @@
+# Elasticsearch
+
+Elasticsearch is the [world's most downloaded vector database](https://www.elastic.co/elasticsearch/vector-database). Elasticsearch offers an efficient way to create, store, and search vector embeddings at scale.
```

Comment:
> world's most downloaded vector database

what is the source of this assertion? perhaps, to maintain the neutrality of the cookbook as a resource, it might be beneficial to refrain from making statements with this tone

Reply:
No worries, was just borrowing from the title of that page for the link text. I'll update text to be neutral.